module/datamanager.py

```diff
@@ -1000,21 +1024,42 @@ def get_overall_it_state(self, user):
         return hosts_state, services_state
 
     def get_important_elements(self, user, type='all', sorter=worse_first):
-        return self.search_hosts_and_services('bi:>2 ack:false type:%s' % type, user=user, sorter=sorter)
+        # BI is greater than the minimum business impact
+        # todo: @maethor: why searching with ack:false ? an element is important whether it is ack or not...
+        search = "bi:>%d " % self.min_business_impact + 'ack:false type:%s' % type
+        return self.search_hosts_and_services(search, user=user, sorter=sorter)
 
-    def get_impacts(self, user, search='is:impact bi:>=0 type:all', sorter=worse_first):
+    def get_impacts(self, user, search='is:impact type:all', sorter=worse_first):
         if "is:impact" not in search:
             search = "is:impact " + search
+        if "bi:" not in search:
+            search = "bi:>=%d " % self.min_business_impact + search
```

Comment:
Please put parenthesis around the % operation, for readability.

Reply:
Ok